translation-concordance.md

```diff
@@ -18,6 +18,27 @@ An automatically-generated list of page translation relationships across this si
   <tr>
     {% for l in site.data.snippets.language-list %}
     {% assign sp = page_versions | where: "lang", l | first %}
+    {% assign where_exp: "item", "item.layout == 'lesson'" %}
+    <td><a href="{{sp.url}}">{{ sp.title }}</a></td>
+    {% endfor %}
+  </tr>
+{% endfor %}
+</table>
+
+{% assign original_pages = site.pages | where_exp: "item", "item.name != 'redirect.html'" | where_exp: "item", "item.name != 'redirects.json'" | where_exp: "item", "item.name != 'index.md'" | where_exp: "item", "item.original == nil" %}
+
+<table>
+  <tr>{% for l in site.data.snippets.language-list %}
+    <th>{{ l }}</th>{% endfor %}
+  </tr>
+  {% for p in original_pages %}
+  {% assign reverse_p = p.name | split: "." | first %}
+  {% assign translated_pages = site.pages | where: "original", reverse_p %}
+  {% assign page_versions =  p | concat: translated_pages %}
+  <tr>
+    {% for l in site.data.snippets.language-list %}
+    {% assign sp = page_versions | where: "lang", l | first %} 
+    {% assign where_exp: "item", "item.layout != 'lesson'" %}
```

Comment:
This is where things go off the rails - it's not where the `where_exp: "item", "item.layout != 'lesson'"` belongs. You want to put it instead up in the `assign` statement on line 28, after `where_exp: "item", "item.original == nil"`.

Everything else inside the `<table>...</table>` tags ought to stay the same. What we're trying to do is create a new version of the `assign` statement in the original code, and then using the exact same table code to loop through just the lessons.